docs/tutorials/02-channel-upgrades/05-upgrade-channel.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Upgrade channel
+sidebar_label: Upgrade channel
+sidebar_position: 5
+slug: /channel-upgrades/upgrade-channel
+---
+
+# Upgrade the ICS 20 transfer channel
+
+## Start the relayer
+
+We start the relayer:
+
+```bash
+hermes --config config.toml start
```

Comment:
I had some problems with specifying configs in the past. Make sure this works in hermes v1.9.0

Reply:
oh, really? I never had any issues with it.